spa/src/pages/world-browser/WorldBrowserTools.vue

```diff
@@ -10,9 +10,13 @@
       <button class="btn-ui"
               v-on:click="opener('#/messageboard/' + $store.data.place.id)">Job Offers</button>
     </span>
+    <span v-else-if="$store.data.place.type === 'shop'">
```

Comment:
Question for community: Did each store have their own inbox and message board, or did all of them link to mall's?